src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -22,6 +22,11 @@ public interface IPreferences
         bool ShowConnector { get; set; }
 
         /// <summary>
+        /// Indicates if preview bubbles should be displayed on nodes.
+        /// </summary>
+        bool ShowPreviewBubbles { get; set; }
```

Comment:
@ikeough @ramramps I don't think this is an issue but just want to bring it up for discussion - these are changes to another public interface, though not one that zero touch users will ever touch, do we need to do something special here or ensure anything before changing interfaces like this in core?